bin/git-authors

```diff
@@ -37,10 +36,10 @@ fi
 authors() {
   if $NO_EMAIL; then
     # email will be used to uniq authors.
-    git shortlog -sne | awk '{$1=""; sub(" ", ""); print}' | awk -F'<' '!x[$1]++' | awk -F'<' '!x[$2]++' \
+    git shortlog HEAD -sne | awk '{$1=""; sub(" ", ""); print}' | awk -F'<' '!x[$1]++' | awk -F'<' '!x[$2]++' \
```

Comment:
Noting that this addition of `HEAD` potentially changes behavior. `git-shortlog` states the following:

> If no revisions are passed on the command line and either standard input is not a terminal or there is no current branch, git shortlog will output a summary of the log read from standard input, without reference to the current repository.

Reply:
It seems like this changes makes the behavior of `git-authors` more intuitive, say, when currently in a detached HEAD state. Was this the original intent behind this change?